packages/odyssey-storybook/applitools.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 
   matchLevel: "Strict",
   showStorybookOutput: true,
-  testConcurrency: 10,
+  testConcurrency: 20,
```

Comment:
unrelated change to a11y checking here to align our concurrency with our account threshold